bindsnet/datasets/alov300.py

```diff
@@ -0,0 +1,318 @@
+from __future__ import print_function, division
+
+import os
+import numpy as np
+import torch
+import shutil
+import zipfile
+import sys
+import time
+import warnings
+import cv2
+import random
+import bindsnet.datasets.preprocess
+
+
+from PIL import Image
+from glob import glob
+from tqdm import tqdm
+from collections import defaultdict
+from typing import Optional, Tuple, List, Iterable
+from urllib.request import urlretrieve
+from torchvision import transforms
+import xml.etree.ElementTree as ET
+from torch.utils.data import Dataset
+
+warnings.filterwarnings("ignore")
+
+
+class ALOV300(torch.utils.data.Dataset):
```

Comment:
Is there a reason for using the whole namespace when it's imported as `Dataset` above?

Reply:
haha, no there is not